res/database/Default/user/scr0llbaer_news.xml

```diff
@@ -2260,5 +2260,1642 @@
+					<en>kettle;an old kettle|egg cooker;an old egg cooker|toaster;an old toaster|sandwich maker;an old sandwich maker|waffle iron;an old waffle iron|blender;an old blender|coffeemaker;an old coffeemaker|espresso machine;an old espresso machine</en>
+					<pl>czajnik;stary czajnik zaczął|jajowar;stary jajowar zaczął|toster;stary toster zaczął|opiekacz do kanapek;stary opiekacz do kanapek zaczął|gofrownica;stara gofrownica zaczęła|mikser;stary mikser zaczął|ekspres do kawy;stary ekspres do kawy zaczął|ekspres do kawy;stary ekspres do kawy zaczął</pl>
+				</kettle_csv>
+				<kettle>
+					<all>${.csv:kettle_csv:0}</all>
+				</kettle>
+				<anoldkettle>
+					<all>${.csv:kettle_csv:1}</all>
+				</anoldkettle>
+			</variables>
+			<data genre="4" price="1" quality_min="5" quality_max="40" fictional="1" />
+		</news>
+
+		<news guid="63a8815d-614a-4c6f-b578-c3db5090fc50" type="0" creator="9551">
+			<title>
+				<de>${amsterdam}: Verirrtes Partyboot</de>
+				<en>${amsterdam}: Party boat gone astray</en>
+				<pl>${amsterdam}: Imprezowa łódź zabłądziła</pl>
+			</title>
+			<description>
+				<de>Ein Partyboot in ${amsterdam} ist ohne Kapitän einen Kanal entlang getrieben. Die feiernden Gäste bemerkten erst nach einer Stunde, dass sie weit ab vom Schuss waren.</de>
+				<en>A party boat in ${amsterdam} drifted down a canal without its captain. Partygoers only noticed an hour later they were far off the beaten track.</en>
+				<pl>${amsterdam}: Łódź imprezowa odpłynęła kanałem bez kapitana. Imprezowicze dopiero po godzinie zauważyli, że znaleźli się daleko od utartego szlaku.</pl>
+			</description>
+			<variables>
+				<amsterdam>
+					<de>Amsterdam|Brügge|Stockholm|Birmingham|Köln</de>
+					<en>Amsterdam|Bruges|Stockholm|Birmingham|Cologne</en>
+					<pl>Amsterdam|Brugia|Sztokholm|Birmingham|Kolonia</pl>
+				</amsterdam>
+			</variables>
+			<data genre="4" price="1" quality_min="5" quality_max="40" fictional="1" />
+		</news>
+
+		<news guid="16362271-77ae-49d0-a95e-4e29c4cbc906" type="0" thread_guid="ebfbed5f-cce7-42c6-935e-b0bde05c09da" creator="9551">
+			<title>
+				<de>Italien erfindet Feiertag</de>
+				<en>Italy invents holiday</en>
+				<pl>Włochy wymyślają święto</pl>
+			</title>
+			<description>
+				<de>Italien führt den „Nationalen Siesta-Tag“ ein, um die Produktivität zu fördern. Kritiker nennen ihn „den faulen Freitag“. Unternehmer klagen über steigende Kissenkosten.</de>
+				<en>Italy introduce "National Siesta Day" to boost productivity. Critics call it "lazy Friday". Businesses complain about rising pillow costs.</en>
+				<pl>Włochy wprowadziły „Narodowy Dzień Sjesty“ dla zwiększenia produktywności. Krytycy nazywają go „leniwy piątek“. Przedsiębiorcy narzekają na rosnące koszty poduszek.</pl>
+			</description>
+			<data genre="0" price="1" quality_min="5" quality_max="40" fictional="1" />
```

Comment:
unique? Without variables this item (and following) might be repeated often.
Another option would be effects ensuring they are not available for a couple of years.

Reply:
it shares the thread id with similar items, which will be my approach in such cases.

Reply:
OK.